apps/mobile/src/features/ImportPrivateKey/components/ImportError/ImportError.tsx

```diff
@@ -0,0 +1,55 @@
+import { Badge } from '@/src/components/Badge/Badge'
+import { Identicon } from '@/src/components/Identicon'
+import { SafeButton } from '@/src/components/SafeButton/SafeButton'
+import { SafeFontIcon } from '@/src/components/SafeFontIcon'
+import { LargeHeaderTitle } from '@/src/components/Title/LargeHeaderTitle'
+import React from 'react'
+import { ScrollView } from 'react-native'
+import { Text, View } from 'tamagui'
+
+export function ImportError() {
+  return (
+    <View flex={1} paddingBottom={80} justifyContent="space-between">
+      <View flex={1}>
+        <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
+          <View flex={1} flexGrow={1} alignItems="center" justifyContent="center" paddingHorizontal="$3">
+            <View flexDirection="row" alignItems="center" gap="$3">
+              <Identicon address="0x0000000000000000000000000000000000000000" size={64} />
```

Comment:
Why are we generating an identicon of the 0x0 address? Later we say - the address is not a signer of this account. I would expect that we generate an identicon from the provided private key

Reply:
Oh noh, this might be a result of the merge that I did, will fix it. thanks for catching it